scripts/py_matter_yamlclusters/BUILD.gn

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("//build_overrides/pigweed.gni")
+import("$dir_pw_build/python.gni")
+
+pw_python_package("matter_yamlclusters") {
+  setup = [
+    "setup.py",
+    "setup.cfg",
+    "pyproject.toml",
+  ]
+
+  sources = [
+    "matter_yamlclusters/__init__.py",
+    "matter_yamlclusters/cluster/delay_commands.py",
+    "matter_yamlclusters/cluster/log_commands.py",
+    "matter_yamlclusters/cluster/system_commands.py",
+    "matter_yamlclusters/clusters.py",
+  ]
+
+  python_deps = [ "${chip_root}/scripts/py_matter_yamltests:matter_yamltests" ]
```

Comment:
If this needs to be a standalone package for something csg wants this should not be dependent on matter_yamltests. In general we should prevent circular dependencies

I guess that would mean bring in the abstract classes into this module

Reply:
That is not a concern anymore :)